Pod/Classes/PINAnimatedImageManager.m

```diff
@@ -41,15 +46,22 @@ + (instancetype)sharedManager;
 
 @implementation PINAnimatedImageManager
 
-+ (void)initialize
++ (void)load
```

Comment:
I've never been able to find documentation on when it's cleared but it's definitely not on every run of the app (I just confirmed to be sure).